modules/ROOT/pages/batch-error-handling-faq.adoc

```diff
@@ -1,23 +1,33 @@
-= Handling Errors During Batch Job
+= Handling Errors During Batch Jobs
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 == Overview
 
-Handling errors inside a batch job can be complicated considering the huge amount of data this scope is designed to take. +
-When processing 1 million records, it's almost impossible to log everything. Logs would become huge, unreadable, and would take a massive toll on the performance of your processing. +
-On the other hand, logging too little would make it impossible for you to know what went wrong. And considering that if out of those 1 million records, 30 thousand of them failed, not knowing what’s wrong with them can turn your debugging task very problematic. This trade-off is not simple to overcome.
+Mule batch processing is designed to handle very large data sets and to perform "near real-time" data integration that recovers from crashes and continues processing a job from a point of failure. However, _verbose_ logs for issues that occur in large data sets can become enormous and severely impact performance. 
```

Comment:
What does "near real-time" mean? Where did you get this description from?

Reply:
@fsgonz It is a phrase that (I believe) Mariano Gonzalez used at some point. Would it make sense to something like "almost real-time" or just leave that out altogether?